lib/init/index.js

```diff
@@ -1,13 +1,91 @@
-export function builder(yargs) {
+import { copy, existsSync } from 'fs-extra';
+import { join } from 'path';
+import chalk from 'chalk';
+import cleanupFiles from './utils/clean-up-files';
+
+const CWD = process.cwd();
+const BIGTEST_DIR = `${CWD}/bigtest`;
+let CLI_TEMPLATE_DIR = join(__dirname, '../../templates');
```

Comment:
I think all top-level things should be `const`. At least that's what I've done for the rest of this repo

Reply:
Yeah, this totally should be a `const` since it's in screaming snake case